### PR TITLE
Remote DB for benchmarks

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Description: remote DB for benchmarks

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: docs/benchmarks.md [hunk 1]
```diff
@@ -24,6 +24,45 @@ Benchmarks can be started from command line by the following command:
 make benchmark
 ```
 
+## Database benchmarks
+
+Database benchmarks can be run against local or remote database. The
+configuration is stored in
+[tests/benchmark.tom](https://github.com/RedHatInsights/ccx-notification-writer/blob/master/tests/benchmark.toml)
```
Comment (juandspy) [inline docs/benchmarks.md:31]: wouldn't it be better to use
```markdown
[tests/benchmark.tom](tests/benchmark.toml)
```
?

Comment (juandspy) [inline docs/benchmarks.md:31]: typo, it should be "toml", not "tom"

**Review by juandspy — APPROVED** [review]

**Review by Bee-lee — APPROVED** [review]
LGTM except for the toml typo

File: docs/benchmarks.md (hunk 1 above)
Comment (tisnik) [reply docs/benchmarks.md:31]: nice catch

Comment (juandspy) [reply docs/benchmarks.md]: it should :thinking: why not?

Comment (juandspy) [reply docs/benchmarks.md]: f.e https://github.com/RedHatInsights/ccx-notification-writer/blob/master/docs/architecture.md?plain=1#L10

**Pull request merged**